src/zcash/JoinSplit.hpp

```diff
@@ -15,6 +15,14 @@
 
 namespace libzcash {
 
+static constexpr size_t GROTH_PROOF_SIZE = (
+    48 + // π_A
+    96 + // π_B
+    48); // π_C
+
+typedef boost::array<unsigned char, GROTH_PROOF_SIZE> GrothProof;
+typedef boost::variant<ZCProof, GrothProof> SproutProof;
```

Comment:
I guess ideally `ZCProof` would have been changed to `BCTVProof` or `PinocchioProof`. This would also force going over all places where `ZCProof` is used.